### PR TITLE
French

### PR DESCRIPTION
**Pull request by shawna-slh**
**[Draft French translation](https://deploy-preview-173--wai-media-guide.netlify.app/media/av/planning/fr)**

**[Published English resource](https://www.w3.org/WAI/media/av/)**

**[changelog](https://www.w3.org/WAI/media/av/changelog/)** with several significant and substantive updates since the draft translation

### CODE REVIEW

File: content/acknowledgements.fr.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ref: /media/av/acknowledgements/   # Do not change this
+layout: default
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/acknowledgements.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/transcribing/
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: Incorrect previous link : `/media/av/player` in the original version

File: content/acknowledgements.fr.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: "Remerciements"
+nav_title: "Remerciements"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: `last_updated`, `translators` & `contributors` lines are missing.

File: content/acknowledgements.fr.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/acknowledgements.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/transcribing/
+
+description: Rendre les contenus audios et vidéos accessibles est développé par le Groupe de travail Éducation et Promotion (EOWG) du Web Accessibility Initiative (WAI) avec le soutien...
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: "Web Accessibility Initiative" needs to be translated. See https://github.com/w3c/translation-glossaries/issues/24
Suggestion : "le groupe de travail Éducation et Promotion (EOWG) de l'Initiative pour l'accessibilité du Web (WAI)"

File: content/acknowledgements.fr.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+image: /content-images/wai-media-guide/social.png
+
+---
+
+<p><cite>Rendre des contenus audios et vidéos accessibles</cite>est développé par le Groupe de travail Education et formation (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Rédigé initialement dans le cadre du projet <a href="https://www.w3.org/WAI/WCAGTA/">WCAG TA</a> financé par le <abbr title="United States">U.S.</abbr> Access Board. Révisé dans le cadre du projet <a href="https://www.w3.org/WAI/expand-access/">WAI Expanding Acess</a> financé par la fondation Ford.</p>
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: * Some mistakes to fix in the first sentence :

Suggestion 

```
<cite>Faire des contenus audios et vidéos accessibles</cite> est développé par le groupe de travail Éducation et Promotion
```

* "It was originally drafted as part of the WCAG TA Project". I guess a closer translation would be "Il fut initialement ébauché dans le cadre du projet WCAG TA"
* I agree we don't need to translate "U.S. Access Board" in this context. As a consequence, we may want to put it between `<i>` tags as it is a foreign phrase. @shawna-slh I don't know if you also want lang or/and class attributes for foreign phrases (see https://www.w3.org/International/questions/qa-b-and-i-tags.en)
* "It was significantly revised as part of the WAI Expanding Access project". A closer translation would be "Il fut significativement révisé dans le cadre du projet WAI Expanding Access.
* @shawna-slh Do you want us to translate "WAI Expanding Access", or treat it as a brand ?

File: content/acknowledgements.fr.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## Auteur
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: "Rédacteur" is probably closer to "Editor"

File: content/acknowledgements.fr.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+* **[Shawn Lawton Henry](https://www.w3.org/People/Shawn/)**
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: The page has been updated with the organizations of each editors/contributors.

File: content/acknowledgements.fr.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+Auteurs précédents :
+* Geoff Freed
+* Judy Brewer
+
+## Contributeurs au EOWG
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: "Contributeurs **du** EOWG" is better don't you think ?

File: content/acknowledgements.fr.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+* Sharron Rush
+* Shadi Abou-Zahra
+* Sylvie Duchateau
+* Vicki Menezes Miller
+
+## Graphiste
+* Ari Stiles
+
+## Crédits photo
+* Photo de l'utilisateur de l'équipement braille par Jordan Nicholson pour <a href="https://disabilityin.org/best-practices/disability-stock-photography/">Disability:IN</a>
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: We could make the list more consistent by not using articles at the beginning of each item.
* Photo de
* Photo de 
* Photo de

File: content/acknowledgements.fr.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+* La photo de la nourriture dans le gril est fournie par <a href="https://www.tripadvisor.com/LocationPhotoDirectLink-g190454-d12929745-i283670458-Beef_glory-Vienna.html#283670458">TripAdvisor</a>
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: * "courtesy of" is generally translated to "avec l'aimable autorisation de"
* Suggestion : "Photo de nourriture sur le grill avec l'aimable autorisation de TripAdvisor"

File: content/acknowledgements.fr.md (hunk 8 above)
Comment (remibetin) [inline content/acknowledgements.fr.md]: Suggestion "Photo d'un utilisateur d'un assistant braille" to be consistent with an existing translation here : 

> "Braille devices with capabilities of small computers incorporate Braille displays." > "Les assistants braille, qui ont la capacité de petits ordinateurs, sont équipés d’afficheurs braille."
https://www.w3.org/WAI/people-use-web/tools-techniques/fr)

File: content/acknowledgements.fr.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+* La photo de la personne qui signe par Jordan Nicholson pour <a href="https://disabilityin.org/best-practices/disability-stock-photography/">Disability:IN</a>
```
Comment (remibetin) [inline content/acknowledgements.fr.md]: Suggestion "Photo d'une personne qui signe"

File: content/av-content.fr.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
+---
+title: "Contenu audio et contenu vidéo"
+title_image: /content-images/wai-media-guide/av-content.svg
+nav_title: "Contenu audio et contenu vidéo"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+last_updated: 2020-10-27   # Change to date of translation YYYY-MM-DD (month in middle)
+# translator: "..."
```
Comment (remibetin) [inline content/av-content.fr.md]: Missing translators & contributors

File: content/captions.fr.md [hunk 12]
```diff
@@ -0,0 +1,256 @@
...
+  previous: /media/av/description/
+  next:     /media/av/transcripts/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Vous aide à comprendre et à créer des légendes (aussi appelées "sous-titres") pour l'accessibilité des contenus audios et vidéos.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 12 septembre 2020. JOURNAL DES MODIFICATIONS.</p>
```
Comment (remibetin) [inline content/captions.fr.md]: We should not translate "CHANGELOG" as it is automatically translated in the process.

File: content/captions.fr.md [hunk 13]
```diff
@@ -0,0 +1,256 @@
...
+   <p><strong>Rédactrice :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```
Comment (remibetin) [inline content/captions.fr.md]: We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.

File: content/player.fr.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+  next: /media/av/description/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Introduit les considérations à prendre en compte pour fournir un lecteur multimédia qui accepte l'accessibilité.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
+   <p><strong>Auteur :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```
Comment (remibetin) [inline content/player.fr.md]: We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.

File: content/player.fr.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+  previous: /media/av/av-content/
+  next: /media/av/description/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Introduit les considérations à prendre en compte pour fournir un lecteur multimédia qui accepte l'accessibilité.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
```
Comment (remibetin) [inline content/player.fr.md]: We should not translate "CHANGELOG" as it is automatically translated in the process.

File: content/sign-languages.fr.md [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+  previous: /media/av/transcripts/
+  next:     /media/av/transcribing/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Vous aide à comprendre et à créer une interprétation en langue des signes pour l'accessibilité des contenus audios et vidéos.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
```
Comment (remibetin) [inline content/sign-languages.fr.md]: We should not translate "CHANGELOG" as it is automatically translated in the process.

File: content/sign-languages.fr.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+   <p><strong>Rédactrice :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```
Comment (remibetin) [inline content/sign-languages.fr.md]: We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.

File: content/transcribing.fr.md [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+  previous: /media/av/sign-languages/
+  next:     /media/av/acknowledgements/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Fournit de l'aide pour transcrire un contenu audio en texte -- pour créer des légendes et des transcriptions pour l'accessibilité des contenus multimédia audios et vidéos.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
```
Comment (remibetin) [inline content/transcribing.fr.md]: We should not translate "CHANGELOG" as it is automatically translated in the process.

File: content/transcribing.fr.md [hunk 19]
```diff
@@ -0,0 +1,151 @@
...
+   <p><strong>Rédacteur :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```
Comment (remibetin) [inline content/transcribing.fr.md]: We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.

File: content/transcripts.fr.md [hunk 20]
```diff
@@ -0,0 +1,316 @@
...
+  previous: /media/av/captions/
+  next:     /media/av/sign-languages/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Vous aide à comprendre et à créer des transcriptions pour rendre les contenus audios et vidéos multimédia accessibles.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 9 octobre 2020. JOURNAL DES MODIFICATIONS.</p>
```
Comment (remibetin) [inline content/transcripts.fr.md]: We should not translate "CHANGELOG" as it is automatically translated in the process.

File: content/transcripts.fr.md [hunk 21]
```diff
@@ -0,0 +1,316 @@
...
+   <p><strong>Rédactrice :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```
Comment (remibetin) [inline content/transcripts.fr.md]: We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.

**Comment by remibetin** [comment]
@shawna-slh I'm still in the process of reviewing but I have already noticed many things that need updates and fixes. Because I can't directly edit your PR, how do you want to proceed ? Should I create PRs to your branch ?

File: content/acknowledgements.fr.md (hunk 2 above)
Comment (shawna-slh) [reply content/acknowledgements.fr.md]: Let's not translate the Acknowledgements page.

**Comment by remibetin** [comment]
Integrated into #177

**Pull request closed without merging**